src/main/java/de/numcodex/feasibility_gui_backend/query/result/ResultService.java

```diff
@@ -20,18 +25,51 @@
 public class ResultService {
 
   private final Cache<Long, QueryResult> queryResultCache;
+  private final BrokerAdmin2 aktinBrokerClient;
```

Comment:
Can this be put in the de.numcodex.feasibility_gui_backend.query.broker.aktin.AktinBrokerClient ? Or maybe added to the BrokerClient Interface for all implementing classes to have.